enhancements/console/quick-starts.md

```diff
@@ -151,6 +154,23 @@ resource actions. In order for the user to see the Quick Start and
 take its tour, his access review needs to pass all the listed
 resources actions.
 
+#### Quick Start Internationalization
+
+OpenShift can be toggled between multiple languages. The
+Quick Start CRD `spec` contains a `language` field to specify what
```

Comment:
As @jhadvig suggests, we might make this a label so we can exclude other languages from the response using a label query. That's the advantage to having each translation being in a separate resource: you don't request QuickStarts for languages that you don't need.

Reply:
Another option is to have a console backend endpoint that filters by `spec.language` if we don't need to watch QuickStarts.

Reply:
Thinking if the label would not be sufficient, since the label would be basically the same as the proposed field.
The only thing is that we will need to have it documented, thats the label is required, otherwise the QS won't be fetched.

Reply:
A missing label could be treated as the default language (English).

I don't think we can require that the label is set since it's effectively API breaking. Existing QuickStarts would disappear on upgrade.

Reply:
I like the idea of using a label so that we can limit the results when fetching resources.